src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
```

Comment:
Don’t use static methods in your solution 

see common mistakes https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery